admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,550 @@
+self-provisioners                               /self-provisioner                                          system:authenticated:oauth
+system:service-serving-cert-controller          /system:service-serving-cert-controller                                                                   openshift-infra/service-serving-cert-controller
+system:registrys                                /system:registry                                           system:registries
+system:pv-binder-controller                     /system:pv-binder-controller                                                                              openshift-infra/pv-binder-controller
+system:build-strategy-jenkinspipeline-binding   /system:build-strategy-jenkinspipeline                     system:authenticated
+system:deployment-controller                    /system:deployment-controller                                                                             openshift-infra/deployment-controller
+system:masters                                  /system:master                                             system:masters
+system:service-load-balancer-controller         /system:service-load-balancer-controller                                                                  openshift-infra/service-load-balancer-controller
+----
+====
+
+These commands can generate huge lists, so you may want to pipe the output into a text file that you can search through more easily.
+
+[[control-user-roles]]
+== Controlling User Permissions with Roles
+
+You can define
+link:../admin_guide/manage_authorization_policy.html[roles] (or permissions)
+for a user before their initial log in so they can start working immediately.
+You can assign many different types of roles to users such as admin,
+basic-user, self-provisioner, and cluster-reader.
+
+For a complete list of all available roles:
+
+----
+$ oadm policy
+----
+
+The following section includes examples of some common operations related to
+adding (binding) and removing roles from users and groups. For a complete list
+of available local policy operations, see
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[Managing Role Bindings].
+
+=== Adding a Role to a User
+
+To bind a role to a user for the current project:
+
+----
+$ oadm policy add-role-to-user <role> <user_name>
+----
+
+You can specify a project with the `-n` flag.
+
+=== Removing a Role from a User
+
+To remove a role from a user for the current project:
+
+----
+$ oadm policy remove-role-from-user <role> <user_name>
+----
+
+You can specify a project with the `-n` flag.
+
+=== Adding a Role to a User for All Projects
+
+To bind a role to a user for all projects in the cluster:
+
+----
+$ oadm policy add-cluster-role-to-user <role> <user_name>
+----
+
+=== Removing a Role from a User for All Projects
+
+To remove a role from a user for all projects in the cluster:
```

Comment:
This only affects the cluster rolebinding.  If three project admins each gave david edit powers in their project, this command wouldn't do anything.